### ISSUE TITLE
crash after "select none" in "create new style"

### ISSUE THREAD

**Issue by bajdero** [darktable-org/darktable#16513]
### Describe the bug

The "select none" button in "create new style" dialog leads to darktable crash. To deselect modules user needs to take them one by one. Problem occurs also when editing saved styles.

Tested on:
darktable: 4.6.1 flatpak from flathub
darktable 4.6.1 rpm from fedoraproject.org
Tested on a machine with Nvidia GPU and without discrete GPU

### Steps to reproduce

1. Go to lighttable
2. Select any photo with any modules active
3. Go to Styles
4. Click Create...
5. Click select none
6. Try toggling on any module or set the name of the style and wait for crash.

### Logfile | Screenshot | Screencast

from var_log_messages, maybe related:

```
mar 24 09:52:45 fedora darktable[55298]: gtk_tree_view_get_model: assertion 'GTK_IS_TREE_VIEW (tree_view)' failed
mar 24 09:52:45 fedora darktable[55298]: gtk_tree_model_get_iter: assertion 'GTK_IS_TREE_MODEL (tree_model)' failed
mar 24 09:52:45 fedora darktable[55298]: gtk_tree_model_get: assertion 'GTK_IS_TREE_MODEL (tree_model)' failed
mar 24 09:52:45 fedora darktable[55298]: gtk_list_store_set_valist: assertion 'GTK_IS_LIST_STORE (list_store)' failed
mar 24 09:52:45 fedora darktable[55298]: gtk_list_store_set_valist: assertion 'GTK_IS_LIST_STORE (list_store)' failed
```
darktable log:
[darktable_bt_N8FPK2.txt](https://github.com/darktable-org/darktable/files/14734935/darktable_bt_N8FPK2.txt)

### Where did you obtain darktable from?

downloaded from www.darktable.org

### darktable version

4.6.1 flathub

### What OS are you using?

Linux

### What is the version of your OS?

Fedora 39

### Are you using OpenCL GPU in darktable?

Yes

### If yes, what is the GPU card and driver?

NVIDIA GeForce RTX 3070, NVIDIA-SMI 550.54.14

### Please provide additional context if applicable. You can attach files too, but might need to rename to .txt or .zip

tested with raw and png files without old edits on clean install of darktable without LUA scripts activated

**Comment by pehar1** [comment]
Not able to reproduce with fresh install of self build 4.7.0+806~gb84a94d477 on Ubuntu 22.04. Don't get the GTK messages.

**Comment by bajdero** [comment]
On darktable-4.7.0~git809.3e9f0ae3-11488.1.x86_64 I also don't have this issue so it probably was mitigated at some point so I need to wait for new release and flatpak container.

**Comment by gi-man** [comment]
Someone commented something similar in pixls.us. https://discuss.pixls.us/t/crash-with-the-style-creation-window/42657

I was not able to recreate on fedora 39 current master.

**Comment by victoryforce** [comment]
Duplicate of #16096 and #16140. Fixed in b27b762934d89207c5518024a5cd563da22dcd00. So I'm closing this issue.
